openapi/components/schemas/OrderItem.yaml

```
@@ -23,6 +23,61 @@ properties:
     anyOf:
       - $ref: ./OriginalPlan.yaml
       - $ref: ./FlexiblePlan.yaml
+  usageLimits:
+    type: object
+    nullable: true
+    description: |-
+      Set the soft and hard usage limits thresholds for an item with metered billing plan.
+      This value is ignored when the plan is not metered.
+    properties:
+      softLimit:
+        type: object
+        nullable: true
+        description: |-
+          Soft limit configures thresholds for notification purposes.
+          When the reported usage reaches 75%, 90% and 100% of the configured limit, a specific event is fired.
+          Webhook and a notification can be configured for this event to notify the customer.
+          Useful for accounting and customer success purposes.
+          Metered billing plan usages can still be reported if the soft limit is reached.
```

suggestion: for readability
```suggestion
          Defines thresholds for notification purposes.
          When the reported usage reaches 75%, 90%, and 100% of the configured limit, a specific event is fired.
          To notify the customer, a webhook and notification can be configured for this event.
          This field is useful for accounting and customer success purposes.
          The usage of metered billing plans can still be reported if the soft limit is reached.
```